packages/legacy/core/App/screens/Scan.tsx

```diff
@@ -36,11 +36,23 @@ const Scan: React.FC<ScanProps> = ({ navigation, route }) => {
   const handleInvitation = async (value: string): Promise<void> => {
     try {
       const connectionRecord = await connectFromInvitation(value, agent)
```

Comment:
Can we rename the top level `connectionRecord` var that comes from `connectFromInvitation`? Just so we're not doing `connectionRecord.connectionRecord`? It's a bit hard to read.